Formula/libgweather.rb

```diff
@@ -1,10 +1,10 @@
 class Libgweather < Formula
   desc "GNOME library for weather, locations and timezones"
   homepage "https://wiki.gnome.org/Projects/LibGWeather"
-  url "https://download.gnome.org/sources/libgweather/40/libgweather-40.0.tar.xz"
-  sha256 "ca4e8f2a4baaa9fc6d75d8856adb57056ef1cd6e55c775ba878ae141b6276ee6"
+  url "https://download.gnome.org/sources/libgweather/4.2/libgweather-4.2.0.tar.xz"
+  sha256 "af8a812da0d8976a000e1d62572c256086a817323fbf35b066dbfdd8d2ca6203"
   license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
-  revision 1
+  version_scheme 1
```

Comment:
https://gitlab.gnome.org/GNOME/libgweather/-/merge_requests/120#note_1286867

Probably needs a livecheck to avoid someone accidentally bumping it back to v40 and having to re-downgrade.

Reply:
hmm, this is interesting, why did they use v40 in the first place?

Reply:
Not sure. I guess they had aligned with GNOME 40 new version scheme but backtracked on that.